tests/testthat/test_add_robust_min_set_objective.R

```diff
@@ -1,19 +1,174 @@
+library(prioritizr)
+library(terra)
+
 test_that("compile (single zone)", {
```

Comment:
If possible, I suggest updating the test to ensure that it returns exactly the correct values. E.g., you could write some R code that produces the expected result and see if the compiled problem has the same output, such as here: https://github.com/prioritizr/prioritizr/blob/main/tests/testthat/test_add_min_shortfall_objective.R#L1-L44